src/main/java/wooteco/subway/dto/LineRequest.java

```diff
@@ -1,7 +1,12 @@
 package wooteco.subway.dto;
 
+import javax.validation.constraints.NotBlank;
+
 public class LineRequest {
+
+    @NotBlank
     private String name;
```

Comment:
```suggestion
    @NotBlank(message = "이름을 입력해주세요.")
    private String name;
```
검증 실패 시 예외에 담을 메세지를 명시해줄수도 있어요. 😎